Automattic/wp-calypso
# Domains: remove sites-list from site-redirect module.

### Testing
1. Go to http://calypso.localhost:3000 and purchase a domain.
2. Go to http://calypso.localhost:3000/domains/manage and select your site with the purchased domain.
3. Pick your domain from the list.
4. Click on `Contacts and Privacy` link.
5. Make sure your contact data displays properly.
6. Click on `Edit Contact Info`.
7. Update your contact details and save them.
8. Wait a bit because there might be some short delay.
9. Click back button, you should see updated data.
10. Purchase Site Redirect (http://calypso.localhost:3000/domains/add/site-redirect/).
11. Go to http://calypso.localhost:3000/domains/manage/.
12. Pick your site.
13. Pick your redirect.
14. Click `Redirect Settings`.
15. Try to update url to new value. It needs to be an existing url (200 HTTP).
16. Go back to http://calypso.localhost:3000/domains/manage/.
17. Pick your domain.
18. Click Transfer Domain.
19. Navigate to different pages and make sure all data loads properly.

### Review
